homeassistant/components/edl21/sensor.py

```diff
@@ -35,7 +36,12 @@
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities,
```

Comment:
Please use this callback typing for the entity addition:

https://github.com/home-assistant/core/blob/ff8affdd049797d364085b391673f8f078cbce75/homeassistant/helpers/entity_platform.py#L65
```suggestion
    async_add_entities: AddEntitiesCallback,
```

Reply:
I looked for but it seems the big majority of integrations do not have a type for async_add_entities.

Anyhow I committed your suggestion.